server/sse.go

```diff
@@ -34,7 +35,7 @@ type sseHandler struct {
 	config        config.Config
 }
 
-func NewSSEHandler(opt Option) sseHandler {
+func NewSSEHandler(opt Option) (sseHandler, error) {
```

Comment:
__[golint]__
* [golint] exported function NewSSEHandler should have comment or be unexported
* [golint] exported func NewSSEHandler returns unexported type server.sseHandler, which can be annoying to use

<!-- Comment by [SideCI](https://sideci.com) -->